apps/bfd-pipeline/bfd-pipeline-rda-grpc/src/main/java/gov/cms/bfd/pipeline/rda/grpc/source/DLQGrpcRdaSource.java

```diff
@@ -216,20 +216,20 @@ Processor dlqProcessingLogic(
                   + startingSequenceNumber,
               e);
         }
+      }
 
-        try {
-          sink.shutdown(Duration.ofMinutes(5));
-        } catch (Exception ex) {
-          if (processResult.getException() != null) {
-            processResult.getException().addSuppressed(ex);
-          } else {
-            processResult.setException(ex);
-          }
+      try {
+        sink.shutdown(Duration.ofMinutes(5));
```

Comment:
this number in minutes seems hidden.
can we make it a static variable ?

Reply:
This is just logic carried over from the existing code, but I can swap that for a constant, as well as the `StandardGrpcRdaSource`, which has the same invocation in it.